controllers/admin/AdminLinkWidgetController.php

```diff
@@ -218,11 +219,45 @@ protected function buildHelper()
         return $helper;
     }
 
-    public function validateForm($data)
+    /**
+     * @param array $fields
+     * @return bool
+     */
+    public function validateLinkName(array $fields)
```

Comment:
The idea is about to prevent having an exception thrown, which currently breaks the form submission.